tests/src/test/resources/kg/context/neuroshapes.json

```diff
@@ -4329,7 +4329,7 @@
     "uHillCoefficient": {
       "@id": "https://bbp.epfl.ch/ontologies/core/parameters/uHillCoefficient"
     },
-    "uberon": "http://purl.obolibrary.org/obo/UBERON_",
```

Comment:
targets ending this way do not work anymore.
Consistent with the Json-Ld playground

Reply:
you mean in underscore?

Reply:
Without a `/` or a `#`